src/components/form/FieldTypeFractional.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  disabled: boolean,
+  displayError: boolean,
+  input: Object,
+  isDirty: boolean,
+  placeholder?: string,
+  setRefForField?: Function,
+}
+
+const NUMERATOR = 0;
+const DENOMINATOR = 1;
+const SPLITTER = ' / ';
+const FieldTypeFractional = ({
+  disabled = false,
+  displayError = false,
+  input: {name, onChange, value},
+  isDirty = false,
+  setRefForField,
+}: Props): React$Node => {
+
+  const handleSetReference = (element: any) => {
+    if(setRefForField) {
+      setRefForField(element);
+    }
+  };
+
+  const changeHandler = (value: string, fieldType: number): void => {
+    const values = value.split(SPLITTER);
+    values[fieldType] = value;
+    onChange(`${values[NUMERATOR] || ''}${SPLITTER}${values[DENOMINATOR] || ''}`);
+  };
```

Comment:
Something here is wonky as this component didn't react to my inputs correctly. When I typed into the numerator field, the denominator field cleared itself, and when I typed into the denominator field, the same value appeared into both of them.

Reply:
Variable name collision.. value is a prop for this component and also a parameter in this changeHandler function... fixed..